src/transformers/integrations/mxfp4.py

```diff
@@ -48,15 +48,16 @@
 
 
 # Copied from GPT_OSS repo and vllm
-def quantize_to_mxfp4(w):
-    downcast_to_mxfp = triton_kernels_hub.numerics_details.mxfp.downcast_to_mxfp
-
-    w, w_scale = downcast_to_mxfp(w.to(torch.bfloat16), torch.uint8, axis=1)
-    w, w_scale = swizzle_mxfp4(w, w_scale)
+def quantize_to_mxfp4(w, triton_kernels_hub):
+    downcast_to_mxfp_torch = triton_kernels_hub.numerics_details.mxfp.downcast_to_mxfp_torch
+    w, w_scale = downcast_to_mxfp_torch(w.to(torch.bfloat16), torch.uint8, axis=1)
```

Comment:
1. we need the torch version
2. swizzle is done at loading time already so duplicating fails